romana-install/roles/cache/apply/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+# OS Files
+- include: "{{ item }}"
+  with_first_found:
+    - files:
+      - "repo_{{ ansible_distribution|lower }}_{{ ansible_distribution_release|lower }}.yml"
```

Comment:
So, "with_first_found" was used in case we ever wish to have more files listed there? Or because of the convenient 'skip' notation?

The distro/release combination was 'calculated' before and is presented in the template variables, was it not? In that case, we will always have a single file.